app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -46,7 +50,63 @@ import com.duckduckgo.app.bookmarks.model.FavoritesRepository
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.HandleNonHttpAppLink
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.HideKeyboard
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.HideWebContent
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.InjectEmailAddress
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.LaunchAddWidget
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.LaunchNewTab
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.LaunchSurvey
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.LaunchTabSwitcher
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.LoadExtractedUrl
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.Navigate
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.NavigateBack
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.NavigateForward
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.OpenAppLink
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.OpenInNewBackgroundTab
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.OpenInNewTab
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.OpenMessageInNewTab
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.Refresh
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.RefreshUserAgent
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.RequestFileDownload
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.RequestSystemLocationPermission
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.RequiresAuthentication
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ResetHistory
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.SaveCredentials
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.SendEmail
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.SendSms
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShareLink
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowAppLinkPrompt
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowDomainHasPermissionMessage
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowEditSavedSiteDialog
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowEmailTooltip
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowErrorWithAction
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowFileChooser
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowFireproofWebSiteConfirmation
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowFullScreen
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowKeyboard
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowSavedSiteAddedConfirmation
+import com.duckduckgo.app.browser.BrowserTabViewModel.Command.ShowWebContent
```

Comment:
I agree with @marcosholgado on this one, changing imports like this adds quite a lot of noise to the PR

Reply:
+1 to @marcosholgado and @joshliebe points here. I think this PR has a ton of noise because it touches imports. Anything related to code-style should be done through spotless and not through bespoke developer's IDE settings.

If the discussion is whether we like star imports or not, and whether we should change them all, that's something we can discuss internally and modify the spotless rules so that it's also done automatically for us.

It's also worth noting that spotless is breaking CI checks here.

Reply:
Sure, I agree that I should delete imports that I'm not using anymore because all the deletions I made, but not modify existing ones through the IDE. I'll change them, thanks! 👍